src/main/java/net/openhft/chronicle/core/threads/EventLoop.java

```diff
@@ -46,23 +46,19 @@
     void unpause();
 
     /**
-     * Notify event loop to stop executing handlers. It is not expected that event loops can then be restarted
+     * Stop executing handlers and block until all handlers are complete.
+     * <p>
+     * It is not expected that event loops can then be restarted.
      */
     void stop();
 
-    /**
-     * @return {@code true} close has been called
-     */
-    @Override
-    boolean isClosed();
```

Comment:
I removed this because it's actually specified in QueryCloseable which is a parent class (perhaps it pre-dated QueryClosable?)